R/server.R

```diff
@@ -614,7 +617,7 @@ server <- function(input, output, session) {
   })
 
   output$peers.table <- renderReactable({
-    df <- get_peers(dea.prod(), ids = selection()[, input$dea_id], threshold = 0)
+    df <- get_peers(dea.prod()$lambda, ids = selection()[, input$dea_id])
```

Comment:
keeping threshold = 0 might be a good idea to avoid changes in the app if the default argument changes in the future (not likely, but possible)

Reply:
Done